packages/core/actions/account-client/getUserOperationGasPrice.ts

```diff
@@ -12,15 +12,34 @@ export const getUserOperationGasPrice = async <
     TChain extends Chain | undefined = Chain | undefined,
     TAccount extends Account | undefined = Account | undefined
 >(
-    client: Client<TTransport, TChain, TAccount, ZeroDevAccountClientRpcSchema>
+    client: Client<TTransport, TChain, TAccount, ZeroDevAccountClientRpcSchema>,
+    provider: string
 ): Promise<Prettify<GetUserOperationGasPriceReturnType>> => {
-    const gasPrice = await client.request({
-        method: "zd_getUserOperationGasPrice",
-        params: []
-    })
+    if (provider === "PIMLICO") {
+        const gasPrice = await client.request({
+            method: "zd_getPimlicoUserOperationGasPrice",
```

Comment:
We can keep the method here `zd_getUserOperationGasPrice` and do the routing in the `meta-aa-provider` using `provider` param from the url.
That way we don't have to change the sdk.